template-helpers/recommendations.js

```diff
@@ -0,0 +1,266 @@
+"use strict";
+
+const { LocaleUtils } = require("./../locale-utils");
+
+
+module.exports = {
+  getAllGenericRecommendations() {
+    return [
+      {
+        recommendationCopy: {
+          subhead: "rec-gen-1-subhead",
+          body: "rec-gen-1",
+          cta: "rec-gen-1-cta",
+        },
+        ctaHref: "https://monitor.firefox.com/security-tips#strong-passwords",
+        ctaShouldOpenNewTab: false,
+        recIconClassName: "rec-gen-1",
+      },
+      {
+        recommendationCopy: {
+          subhead: "rec-gen-2-subhead",
+          body: "rec-gen-2",
+          cta: "rec-gen-2-cta",
+        },
+        ctaHref: "https://monitor.firefox.com/security-tips#five-myths",
+        ctaShouldOpenNewTab: false,
+        recIconClassName: "rec-gen-2",
+      },
+      {
+        recommendationCopy: {
+          subhead: "rec-gen-3-subhead",
+          body: "rec-gen-3",
+          cta: "rec-gen-3-cta",
+        },
+        ctaHref: "https://monitor.firefox.com/security-tips#steps-to-protect",
+        ctaShouldOpenNewTab: false,
+        recIconClassName: "rec-gen-3",
+      },
+      {
+        recommendationCopy: {
+          subhead: "rec-gen-4-subhead",
+          body: "rec-gen-4",
+        },
+        recIconClassName: "rec-gen-4",
+      },
+    ];
+  },
+  getAllPriorityDataClasses(isUserBrowserFirefox=false, isUserLocaleEnUs=false, changePWLink=null) {
+    return  {
+      "government-issued-ids" : {
+        weight: 101,
+        pathToGlyph: "svg/glyphs/social-security-numbers",
+      },
+      "social-security-numbers" : {
+        weight: 101,
+        pathToGlyph: "svg/glyphs/social-security-numbers",
+        recommendations: isUserLocaleEnUs ? [
+          {
+            recommendationCopy: {
+              subhead: "rec-ssn-cta",
+              cta: "rec-ssn-cta",
+              body: "rec-ssn",
+            },
+            ctaHref: "https://www.annualcreditreport.com/index.action",
+            ctaShouldOpenNewTab: true,
+            recIconClassName: "rec-ssn",
+          },
+        ] : "",
```

Comment:
This feels a bit funny. This will either return an array of object, or an empty string?